### ISSUE TITLE
can the loss or some information during training be shown?

### ISSUE THREAD

**Issue by 823858275** [UKPLab/sentence-transformers#510]
i use model.fit() to train the model, but it doesn't show the detail of training, such as loss. I don't find some parameter to show those information.can the loss or some information during training be shown?

**Comment by nreimers** [comment]
You can pass an evaluator to the fit method, as shown here:
https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/nli/training_nli.py

The evaluator is called every `evaluation_steps` and computes whatever scores you like. Usually I use it to compute scores (like accuracy, correlation) on some held-out dev set.

**Comment by iamlockelightning** [comment]
Hi, @nreimers . Can it just return the training loss values during `fit`? (On progress bar, or logging to file/std, or Tensorboard support, ...)

I don't have a held-out dev set. Just want to monitor the training process.

**Comment by Rj7** [comment]
@nreimers This would really useful for Unsupervised learning.
